modules/apps/item-selector/item-selector-test/src/testIntegration/java/com/liferay/item/selector/test/taglib/test/ItemSelectorRepositoryEntryBrowserUtilTest.java

```diff
@@ -73,6 +75,45 @@ public void testAddPortletBreadcrumbEntries() throws Exception {
 
 		DLTestUtil.addDLFileEntry(dlFolder.getFolderId());
 
+		List<BreadcrumbEntry> breadcrumbEntries = _getBreadcrumbEntries(
+			dlFolder.getFolderId());
+
+		Assert.assertEquals(
+			breadcrumbEntries.toString(), 3, breadcrumbEntries.size());
+
+		BreadcrumbEntry breadcrumbEntry = breadcrumbEntries.get(2);
+
+		Assert.assertEquals(dlFolder.getName(), breadcrumbEntry.getTitle());
+	}
+
+	@Test
+	public void testCompanyGroupBreadcrumb() throws Exception {
+		Group controlPanelGroup = GroupLocalServiceUtil.getGroup(
+			_company.getCompanyId(), "Control Panel");
+
+		long originalScopeGroupId = _themeDisplay.getScopeGroupId();
+
+		try {
+			_themeDisplay.setScopeGroupId(controlPanelGroup.getGroupId());
+
+			List<BreadcrumbEntry> breadcrumbEntries = _getBreadcrumbEntries(0);
+
+			Group companyGloablGroup = _company.getGroup();
+
+			BreadcrumbEntry breadcrumbEntry = breadcrumbEntries.get(1);
+
+			Assert.assertEquals(
+				companyGloablGroup.getName(_themeDisplay.getLanguageId()),
+				breadcrumbEntry.getTitle());
+		}
+		finally {
+			_themeDisplay.setScopeGroupId(originalScopeGroupId);
+		}
+	}
+
+	private List<BreadcrumbEntry> _getBreadcrumbEntries(long folderId)
+		throws Exception {
+
 		User user = UserTestUtil.addUser();
```

Comment:
hey @SamZiemer ,
Do we need to instantiate a User for both tests? or could we create a single one during setUp()?

Reply:
Also, shouldn't this user be set for the themeDisplay, not `_themeDisplay.setUser(TestPropsValues.getUser());`?